base_user_role/models/role.py

```diff
@@ -39,6 +57,20 @@ def _compute_user_ids(self):
         for role in self:
             role.user_ids = role.line_ids.mapped("user_id")
 
+    @api.depends("implied_ids", "implied_ids.model_access")
+    def _compute_model_access_ids(self):
+        for rec in self:
+            access = rec.implied_ids.model_access.ids
+            rec.model_access_ids = [(6, 0, access)]
+            rec.model_access_count = len(access)
```

Comment:
shorter (yoiu can do the same for rules)
```suggestion
            rec.model_access_ids = rec.implied_ids.model_access.ids
            rec.model_access_count = len(rec.model_access_ids)
```

Reply:
done